python/plugins/processing/algs/qgis/postgis_utils.py

```diff
@@ -153,7 +153,7 @@ def __init__(self, host=None, port=None, dbname=None, user=None,
         try:
             self.con = psycopg2.connect(self.con_info())
         except psycopg2.OperationalError as e:
-            raise DbError(unicode(e))
+            raise DbError(str(e))
```

Comment:
right :)

can you elaborate why `unicode` is wrong and `str` is correct? `str` is an unencoded byte array in python 2 while the error here should IMHO be an encoded string? Would `repr(e)` or `e.message` work instead of the current approach?